### PR TITLE
Fix logging issues on production.

### PR DESCRIPTION
**Pull request by Qubad786**
## [EDUCATOR-1960](https://openedx.atlassian.net/browse/EDUCATOR-1960)

### CODE REVIEW

**Comment by Qubad786** [comment]
@mushtaqak @muzaffaryousaf have a look.

File: control/veda_val.py [hunk 1]
```diff
@@ -36,6 +34,8 @@
 from control_env import *
 from control.veda_utils import ErrorObject, Output
 
+LOGGER = logging.getLogger(__name__)
```
Comment (mushtaqak) [inline control/veda_val.py]: We may need to add a comment that why we need to add the log here for future references.

File: VEDA/settings/utils.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ def get_logger_config(log_dir='/var/tmp',
         },
         'handlers': {
             'console': {
-                'level': 'DEBUG' if debug else 'INFO',
+                'level': 'INFO',  # TODO: Should be changed to -> 'DEBUG' if debug else 'INFO'
```
Comment (mushtaqak) [inline VEDA/settings/utils.py]: @Qubad786 Can you look at the [logger settings](https://github.com/edx/edx-platform/blob/039afc2517c0fc70af8acb1a33b29c0809e503a4/openedx/core/lib/logsettings.py#L11) from edx-platform

File: VEDA/settings/production.py [hunk 3]
```diff
@@ -13,6 +13,7 @@
 
 CONFIG_DATA = get_config()
 
+# TODO eliminate 'debug=True' once common handler is fixed.
```
Comment (muhammad-ammar) [inline VEDA/settings/production.py]: todo needs to be removed

**Comment by muhammad-ammar** [comment]
lgtm 👍

**Pull request merged**